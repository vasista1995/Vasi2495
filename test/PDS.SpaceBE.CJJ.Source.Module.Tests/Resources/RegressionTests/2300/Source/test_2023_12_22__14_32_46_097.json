{
  "Machine": "LAW033",
  "WireSize": "350",
  "WireType": "Al",
  "Classification": "Routine check(Daily SPC)",
  "Operator": "30156566",
  "ParameterClass": "PPAP",
  "Segment": "IFPS",
  "Module": "MiniFP",
  "Package": "CIPOS MINI",
  "Device": "IKCM15F60GA",
  "FourDReport": "*",
  "SpecialCharacteristics": "ISC",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031310,
      "Seqnr": 4,
      "Value": 1637.4,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031310,
      "Seqnr": 5,
      "Value": 1924.6,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031310,
      "Seqnr": 3,
      "Value": 1524.1,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031310,
      "Seqnr": 2,
      "Value": 1713.7,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 243031310,
      "Seqnr": 1,
      "Value": 1665.2,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "MaterialType": "Schweizer",
  "UniqueSourceKey": "CJJ:BE:2Z3490HU:::Force_ShearOnIGBT:50245:243031310:0",
  "PKey": "CJJ:BE:2Z3490HU:::Force_ShearOnIGBT:50245:243031310:0",
  "ParameterName": "Force_ShearOnIGBT",
  "ParameterUnit": "g",
  "SampleDate": "2023-12-10T22:52:17Z",
  "SampleDateUtc": "2023-12-10T13:52:17Z",
  "SampleSize": 5,
  "Sigma": "147,019097398943",
  "Mean": "1693",
  "Median": "1665,2",
  "Min": "1524,1",
  "Max": "1924,6",
  "Q1": "1637,4",
  "Q3": "1713,7",
  "LdsID": "2300",
  "IsCalcParam": 0,
  "MeanCntrlLow": 1366.642331636,
  "MeanCntrlTarget": 1870.95563492064,
  "MeanCntrlHigh": 2375.26893820527,
  "RangeCntrlHigh": 897.403127094952,
  "SigmaCntrlHigh": 381.989616791615,
  "SpecLow": 550.0,
  "SpecTarget": 1500.0,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "N",
  "CtrlSigmaCenterEnabled": "N",
  "CtrlRangeCenterEnabled": "N",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "N",
  "CreatedDate": "2023-12-10T22:52:17Z",
  "CreatedDateUtc": "2023-12-10T13:52:17Z",
  "UpdatedDate": "2023-12-10T22:52:17Z",
  "UpdatedDateUtc": "2023-12-10T13:52:17Z",
  "RvStoreFlag": "Y",
  "Lot": "2Z3490HU",
  "ChannelId": "50245",
  "CkcId": "0",
  "ChannelName": "Wedge_ShearForce/MiniFP/Al/350/IGBT",
  "SpaceInstanceName": "BE",
  "Sitekey": "CJJ",
  "CalcCreated": "09/17/2023 14:12:58",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 1862.56571428571,
  "ExtEwmaMV": 1805.63012850391,
  "ExtMsMV": 99.9566145885303,
  "OutOfOrder": "N",
  "EwmaS": 147.921306416863,
  "EwmaR": 393.580188740895,
  "Calc2ID": 601756,
  "LimitEnable": "YYNYYNYNNYYYYYYYYYYNNNYYYNYYNNNYNNNNN",
  "ExtEwmaMVLCL": 1702.77499720367,
  "ExtEwmaMVCenter": 1870.94496985084,
  "ExtEwmaMVUCL": 2039.11494249801,
  "ExtMaMVLCL": 1743.82041967572,
  "ExtMaMVCenter": 1870.94496985084,
  "ExtMaMVUCL": 1998.06952002596,
  "EwmaSLCL": 12.9461457853283,
  "EwmaSCenter": 118.42377923198,
  "EwmaSUCL": 223.901412678632,
  "EwmaRLCL": 33.9974850094935,
  "EwmaRCenter": 289.494248314161,
  "EwmaRUCL": 544.991011618829,
  "ExtMSMVLCL": 0.177666655959515,
  "ExtMSMVCenter": 83.7822608207228,
  "ExtMSMVUCL": 336.556586327711,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 1000.0,
  "RawUal": 2620.13994885924,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChannelType": "N",
  "ChState": "Online"
}