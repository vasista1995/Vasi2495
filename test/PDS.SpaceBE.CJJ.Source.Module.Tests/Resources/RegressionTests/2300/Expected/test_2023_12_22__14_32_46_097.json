{
  "IdSource": "SPACEACT2:CJJ:BE:2Z3490HU:::Force_ShearOnIGBT:50245:243031310:0",
  "Data_flat_Metadata": {
    "Machine": "LAW033",
    "Classification": "Routine check(Daily SPC)",
    "Operator": "30156566",
    "Package": "CIPOS MINI",
    "WireSize": "350",
    "ParameterClass": "PPAP",
    "Segment": "IFPS",
    "Module": "MiniFP",
    "Device": "IKCM15F60GA",
    "FourDReport": "*",
    "SpecialCharacteristics": "ISC",
    "WireType": "Al",
    "MaterialType": "Schweizer",
    "Lot": "2Z3490HU",
    "ParameterName": "Force_ShearOnIGBT",
    "ParameterUnit": "g",
    "SiteKey": "CJJ",
    "ChannelId": "50245",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-12-10T13:52:17Z",
    "CreatedTimestamp": "2023-12-10T22:52:17Z",
    "UpdatedTimestamp": "2023-12-10T22:52:17Z",
    "UpdatedTimestampUtc": "2023-12-10T13:52:17Z",
    "ExportedTimestampUtc": "2023-12-10T22:52:17Z",
    "ExportedTimestamp": "2023-12-10T22:52:17Z",
    "SampleTimestampUtc": "2023-12-10T13:52:17Z",
    "SampleTimestamp": "2023-12-10T22:52:17Z",
    "SourceDataLevel": "C",
    "ChannelName": "Wedge_ShearForce/MiniFP/Al/350/IGBT",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/17/2023 14:12:58",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 601756,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2300",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 550.0,
        "SpecTarget": 1500.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYNNYYYYYYYYYYNNNYYYNYYNNNYNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "1693",
        "Median": "1665,2",
        "Sigma": "147,019097398943",
        "Min": "1524,1",
        "Max": "1924,6",
        "Q1": "1637,4",
        "Q3": "1713,7"
      },
      "ControlLimits": {
        "ctrl_mean_low": 1366.642331636,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 1870.95563492064,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 2375.26893820527,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_high": 381.989616791615,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_high": 897.403127094952,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 1862.56571428571,
        "ExtEwmaMV": 1805.63012850391,
        "ExtMsMV": 99.9566145885303,
        "EwmaS": 147.921306416863,
        "EwmaR": 393.580188740895,
        "ExtEwmaMVLCL": 1702.77499720367,
        "ExtEwmaMVCenter": 1870.94496985084,
        "ExtEwmaMVUCL": 2039.11494249801,
        "ExtMaMVLCL": 1743.82041967572,
        "ExtMaMVCenter": 1870.94496985084,
        "ExtMaMVUCL": 1998.06952002596,
        "EwmaSLCL": 12.9461457853283,
        "EwmaSCenter": 118.42377923198,
        "EwmaSUCL": 223.901412678632,
        "EwmaRLCL": 33.9974850094935,
        "EwmaRCenter": 289.494248314161,
        "EwmaRUCL": 544.991011618829,
        "ExtMSMVLCL": 0.177666655959515,
        "ExtMSMVCenter": 83.7822608207228,
        "ExtMSMVUCL": 336.556586327711,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 1000.0,
        "RawUal": 2620.13994885924,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 243031310,
        "Seqnr": 4,
        "Value": 1637.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031310,
        "Seqnr": 5,
        "Value": 1924.6,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031310,
        "Seqnr": 3,
        "Value": 1524.1,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031310,
        "Seqnr": 2,
        "Value": 1713.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031310,
        "Seqnr": 1,
        "Value": 1665.2,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CJJ:BE:::Force_ShearOnIGBT:50245",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:2Z3490HU",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CJJ:BE:2Z3490HU:::Force_ShearOnIGBT:50245:243031310:0"
  }
}